sky/provision/kubernetes/network.py

```diff
@@ -0,0 +1,76 @@
+import os
+from typing import Any, Dict, List, Optional, Tuple
+
+from sky.provision.kubernetes import network_utils
+
+
+def open_ports(
+    cluster_name_on_cloud: str,
+    ports: List[str],  # pylint: disable=unused-argument
```

Comment:
nit: `del ports` for unused argument. Also, shall we use this argument instead of `provider_config['ports']` to align the current implementation for other clouds? Please refer to the comment under `kubernetes-ray.yml.j2`

Reply:
Addressed in https://github.com/skypilot-org/skypilot/pull/2713/commits/186193cf7e82f890dbe6a704d644e48b1e63a788

Reply:
remove the `# pylint: disable=unused-argument`?